src/components/CorporateCampaign/CampaignVerificationForm.vue

```diff
@@ -1,22 +1,35 @@
 <template>
 	<section class="campaign-status section row align-center">
 		<div class="small-12 large-8 align-self-middle columns">
-			<div v-if="iFrameVisible">
+			<kv-lightbox
+				:visible="iFrameVisible"
+				class="employee-verification"
+				@lightbox-closed="close"
+			>
 				<iframe
 					id="faForm"
 					:src="iFrameSrc"
 					:height="iFrameHeight"
 					:width="iFrameWidth"
 					frameborder="0"
 				></iframe>
-				<script src="//kiva.tfaforms.net/js/iframe_resize_helper.js"></script>
-			</div>
+			</kv-lightbox>
 		</div>
 	</section>
 </template>
 
 <script>
+import KvLightbox from '@/components/Kv/KvLightbox';
+
 export default {
+	metaInfo: {
+		script: [
+			{ src: '//kiva.tfaforms.net/js/iframe_resize_helper.js', async: true }
```

Comment:
@eddieferrer @ryan-ludwig verified this works fine with actual form loading and submission